### PR TITLE
AGR-1709: fixed method getOrthologyMultiGeneJson to filter out orthol…

### PR DESCRIPTION
**Pull request by prmani**
…ogs based on stringencyFilter. default=stringent.

### CODE REVIEW

File: agr_java_core/src/main/java/org/alliancegenome/core/service/OrthologyService.java [hunk 1]
```diff
@@ -202,15 +224,19 @@ private static void putGeneInfo(Map<String, Object> map, Gene gene) {
...
-    public static JsonResultResponse<OrthologView> getOrthologyGenes(List<String> geneIDList, OrthologyFilter filter) {
+    public static JsonResultResponse<OrthologView> getOrthologyGenes(List<String> geneIDList, OrthologyFilter orthoFilter) {
         GeneCacheRepository repo = new GeneCacheRepository();
         List<OrthologView> orthologViewList = repo.getAllOrthologyGenes(geneIDList);
         List<OrthologView> filteredOrthologViewList = orthologViewList;
-        if (filter.getStringency() != null && !filter.getStringency().equals(OrthologyFilter.Stringency.ALL)) {
-            filteredOrthologViewList = orthologViewList.stream()
-                    .filter(orthologView -> orthologView.getStringencyFilter().equalsIgnoreCase(filter.getStringency().name()))
+
+        System.out.println("Filter: " + orthoFilter.getStringency().name());
```
Comment (oblodgett) [inline agr_java_core/src/main/java/org/alliancegenome/core/service/OrthologyService.java]: We use a logger for outputting info from the applications... please change the System.out.println -> log.info or log.debug thanks.

Comment (oblodgett) [reply agr_java_core/src/main/java/org/alliancegenome/core/service/OrthologyService.java]: In order to get a logger into the class just add the following annotation to the class:

`@Log4j2`
`public class OrthologyService {`

Comment (prmani) [reply agr_java_core/src/main/java/org/alliancegenome/core/service/OrthologyService.java]: I had used that merely for my own debugging purposes ..I took it out (but accidentally merged my pr :(

Comment (oblodgett) [reply agr_java_core/src/main/java/org/alliancegenome/core/service/OrthologyService.java]: Its all good...

**Pull request merged**